### ISSUE TITLE
Improvements to the plugin admin area

### ISSUE THREAD

**Issue by Poffen** [featherbb/featherbb#122]
It would be nice (and more intuitive) if the list of official plugins shown to the user in the plugin admin area does not contain plugins which the user has already installed. This would make it easier for the user to manage the plugins. I imagine that filtering could be based on plugin name, as any change of plugin version would be taken care of by the update function. What do you think about this?

Putte

**Comment by beaver-dev** [comment]
Why not, either hide them or display the Deactivate/Uninstall button as if it were on the other plugins list.
Maybe we should also move the officiel plugins list into a separate admin menu item (e.g "Marketplace", it would take less time loading the Plugins page, and as one day or the other we also will have themes to display... A search toolbar will also be added, after a few upgrades to the marketplace site!

**Comment by Poffen** [comment]
Yes, I think it's a good idea to separate the official plugins from the ones that are already installed, and place them under a separate menu item. That would look much cleaner IMO. And a search option would also be nice. :)

Putte
